src/API/WesternStatesWater.WestDaat.Contracts.Client/Requests/Admin/OrganizationLoadAllRequest.cs

```diff
@@ -0,0 +1,8 @@
+namespace WesternStatesWater.WestDaat.Contracts.Client.Requests.Admin;
+
+/// <summary>
+/// Request to load all organizations for global admins.
```

Comment:
Slight change to the comment so that it's always true, even if the roles/permissions change some day

```suggestion
/// Request to load all organizations reguardless of organization membership.
```